test/command/cmd.go

```diff
@@ -150,12 +150,18 @@ func (s *Starter) Supernode(running time.Duration, args ...string) (
 	}
 
 	if err = check("localhost", environment.SupernodeListenPort, 10*time.Second); err != nil {
-		s.Kill(cmd)
+		err = s.Kill(cmd)
+		if err != nil {
```

Comment:
does this place necessary? the return is same

Reply:
@yunfeiyanggzq 
I think I can rename the second error,avoid overwriting value.
what do you think?
or I delete this change,keep it the same as before.

Reply:
I think it's better to update the first err. WDYT?

In addition, could you please update the scope of the err which means update the code as follows:

```
// the origin code
       if err = check("localhost", environment.SupernodeListenPort, 10*time.Second); err != nil {
		s.Kill(cmd)
		return nil, err
	}

// updated code
	if err := check("localhost", environment.SupernodeListenPort, 10*time.Second); err != nil {
		s.Kill(cmd)
		return nil, err
	}
```